manifests/apache/vhost.pp

```diff
@@ -81,14 +81,14 @@
 #   No default
 class puppetboard::apache::vhost (
   String $vhost_name,
-  String $wsgi_alias                        = '/',
-  Integer $port                             = 5000,
+  Stdlib::Absolutepath $wsgi_alias          = '/',
```

Comment:
Technically I think `Stdlib::Unixpath` is more correct. It shouldn't accept Windows paths since it's the HTTP path which behaves the same as a unix path.

Reply:
updated it